pynucastro/networks/rate_collection.py

```diff
@@ -2032,13 +2040,30 @@ def create_network_graph(self, node_nuclei, *,
             else:
                 G.labels[n] = fr"${n.pretty}$"
 
+        # use the difference of forward - reverse rate for the graph
+        if use_net_rate:
+            for rp in self.get_rate_pairs():
+                if rp.forward is not None and rp.reverse is not None:
+                    net_rate = rate_ydots[rp.forward] - rate_ydots[rp.reverse]
+                    if normalize_net_rate:
+                        net_rate /= (rate_ydots[rp.forward] + rate_ydots[rp.reverse])
```

Comment:
probably doesn't matter, but maybe its better to divide by `0.5 * (rate_ydots[rp.forward] + rate_ydots[rp.reverse])` so that its consistent with Eq. 12 in https://arxiv.org/pdf/1912.06151, which is what we use.

Reply:
fixed